### PR TITLE
Fixes to work with Click v5

### PR DESCRIPTION
**Pull request by sudorandom**
Adds requirement on click >= 5.

All commands should no longer return the final result. They should output using env.out or env.fout (formatted out).

### CODE REVIEW

**Comment by sudorandom** [comment]
@underscorephil this is ready to review. I recommend you go over each file under SoftLayer/CLI and verify that there are no return statements inside of functions with the `@click.command()` decorator.

File: SoftLayer/CLI/subnet/lookup.py [hunk 1]
```diff
@@ -57,4 +57,4 @@ def cli(env, ip_address):
         device_table.add_row(['name', device['fullyQualifiedDomainName']])
         device_table.add_row(['type', device_type])
         table.add_row(['device', device_table])
-    return table
+    env.fout(table)
```
Comment (underscorephil) [inline SoftLayer/CLI/subnet/lookup.py]: Return still present at line 23

File: SoftLayer/CLI/subnet/create.py [hunk 2]
```diff
@@ -63,4 +63,4 @@ def cli(env, network, quantity, vlan_id, ipv6, test):
             table.add_row([price['item']['description'], rate])
 
     table.add_row(['Total monthly cost', "%.2f" % total])
-    return table
+    env.fout(table)
```
Comment (underscorephil) [inline SoftLayer/CLI/subnet/create.py]: Return present at line 52

File: SoftLayer/CLI/loadbal/service_edit.py [hunk 3]
```diff
@@ -46,4 +46,4 @@ def cli(env, identifier, enabled, port, weight, healthcheck_type, ip_address):
                      port=port,
                      weight=weight,
                      hc_type=healthcheck_type)
-    return 'Load balancer service %s is being modified!' % identifier
+    env.fout('Load balancer service %s is being modified!' % identifier)
```
Comment (underscorephil) [inline SoftLayer/CLI/loadbal/service_edit.py]: Return at line 33

File: SoftLayer/CLI/loadbal/group_edit.py [hunk 4]
```diff
@@ -38,4 +38,4 @@ def cli(env, identifier, allocation, port, routing_type, routing_method):
                            routing_type=routing_type,
                            routing_method=routing_method)
 
-    return 'Load balancer service group %s is being updated!' % identifier
+    env.fout('Load balancer service group %s is being updated!' % identifier)
```
Comment (underscorephil) [inline SoftLayer/CLI/loadbal/group_edit.py]: Return at line 32

File: SoftLayer/CLI/firewall/add.py [hunk 5]
```diff
@@ -54,4 +54,4 @@ def cli(env, target, firewall_type, high_availability):
     elif firewall_type == 'server':
         mgr.add_standard_firewall(target, is_virt=False)
 
-    return "Firewall is being created!"
+    env.fout("Firewall is being created!")
```
Comment (underscorephil) [inline SoftLayer/CLI/firewall/add.py]: Return at line 39

**Comment by underscorephil** [comment]
dns/zone_import.py:42 has a return

**Comment by underscorephil** [comment]
@sudorandom  - Identified a few return stragglers and tested a variety of commands. Seems to be working properly.

**Comment by sudorandom** [comment]
@underscorephil the return in dns/zone_import.py is fine. I believe I've addressed all the other ones that were missed.

**Comment by sudorandom** [comment]
@underscorephil this should be ready to go if you're good with it

**Pull request merged**